integration/client/go.mod

```diff
@@ -15,9 +15,26 @@ require (
 	github.com/opencontainers/image-spec v1.0.1
 	github.com/opencontainers/runtime-spec v1.0.3-0.20200929063507-e6143ca7d51d
 	github.com/pkg/errors v0.9.1
-	github.com/sirupsen/logrus v1.8.0
+	github.com/sirupsen/logrus v1.7.0
 	golang.org/x/sys v0.0.0-20210124154548-22da62e12c0c
 	gotest.tools/v3 v3.0.3
 )
 
-replace github.com/containerd/containerd => ../../
+replace (
+	// use the containerd module from this repository instead of downloading
+	//
+	// IMPORTANT: this replace rule ONLY replaces containerd itself; dependencies
+	// in the "require" section above are still taken into account for version
+	// resolution if newer.
+	github.com/containerd/containerd => ../../
+
+	// Replace rules below must be kept in sync with the main go.mod file at the
+	// root, because that's the actual version expected by the "containerd/containerd"
+	// dependency above.
+	github.com/gogo/googleapis => github.com/gogo/googleapis v1.3.2
```

Comment:
Looks like  we need more effort to sync the version between two go.mod. sad :(

Reply:
> github.com/containerd/containerd v1.5.0-beta.3

could we possible to change it to `github.com/containerd/containerd v0.0.0`. In main branch, the tag doesn't make senses.

Reply:
https://github.com/containerd/containerd/blob/master/integration/client/benchmark_test.go#L17

and I think we need to change the package name from `containerd` to `client`.

Reply:
> could we possible to change it to github.com/containerd/containerd v0.0.0. In main branch, the tag doesn't make senses.

I tried replacing, but it became a bit messy; I also tried changing it to version v1.0.0 (to better show "hey, this version is bogus"), but Microsoft/hcsshim/test depends on containerd/containerd v1.5.0-beta.4, so go mod changed it back to v1.5.0-beta.4.

> and I think we need to change the package name from `containerd` to `client`.

yes, makes sense; I pushed a commit to rename it